Notepad/HomeViewController.swift

```diff
@@ -29,12 +29,8 @@ class ViewController: UIViewController {
     
     @IBAction func handleLoadButtonPressed(_ sender: Any) {
         
-        titleTextField.text =
-            UserDefaults.standard.object(forKey:
-                                            "Title") as? String
+        titleTextField.text = UserDefaults.standard.object(forKey: "Title") as? String
         
-        notesTextView.text =
-            UserDefaults.standard.object(forKey:
-                                            "Body") as? String
+        notesTextView.text =  UserDefaults.standard.object(forKey: "Body") as? String
```

Comment:
I don't think `notesTextView` is a good name, maybe `bodyTextView`? I mean.. you named the UserDefaults key as `"Body"` as well anyways - it will make your code more consistent.